### ISSUE TITLE
Gilcrease Subjects

### ISSUE THREAD

**Issue by VladimirAlexiev** [american-art/AAT-Term-Mappings#1]
`Gilcrease AAC Role Mapping (8.2.16).xlsx` maps role Subjects to 
- aat:300404593 various subjects (information indicator) "used when a work is about multiple subjects, rather than a single coherent subject."

This is wrong because it says "this work has too many subjects to describe, or I don't care to describe it precisely". But I doubt the Gilcrease role has that meaning. Map to the parent
- aat:300404126 subjects (content of works) "In the visual arts, oral works, music, performing arts, and literature, the topics, themes, or other visual, oral, or written content of the work. It includes what the work is of or about."

**Comment by VladimirAlexiev** [comment]
`NPG Roles.xlsx` correctly maps
- Associated Person/Institution -> subjects (content of works)

**Comment by kateblanch** [comment]
Good catch @VladimirAlexiev. I will ask Gilcrease to change this in the source file.

**Comment by kateblanch** [comment]
@diana-folsom

**Comment by Diana-Folsom** [comment]
Thanks!
